tests/cache/tests.py

```diff
@@ -88,28 +89,55 @@ def test_simple(self):
         cache.set("key", "value")
         self.assertIsNone(cache.get("key"))
 
+    async def test_simple_async(self):
+        "Dummy cache backend ignores cache set calls"
+        await cache.aset("key", "value")
+        self.assertIsNone(await cache.aget("key"))
```

Comment:
I think we should move all async tests to a new file, e.g. `tests_async.py` with `AsyncDummyCacheTests`.